### PR TITLE
Features/hdf 200 vesting modal

### PR DESCRIPTION
**Pull request by schystz**
## Description of Changes

- Added Harvest modal on Vesting page
  - shows up after clicking Harvest button from Farm page
  - won't show if simply visiting the Vesting page (click Vesting from main nav)
- On Farm page, renamed `Unclaimed HALO Earned` to `My HALO Earned`

UI tweaks
- Standard border radius of modal screens
- Standard border radius of Halo buttons

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDF-241)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDF-XXX-description`
* [x] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [x] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

**Review by tracyarciaga — APPROVED** [review]
All changes are good. Approving this.

**Comment by tracyarciaga** [comment]
@schystz I renamed instances of dessert pool and DSRT to rainbow pool and RNBW, can I just push it in this  PR too?

**Comment by schystz** [comment]
> All changes are good. Approving this.

Hi @tracyarciaga , do you mind if you push that instead to my working branch `features/hdf-77-harvest-rewards`? I have a feeling that your changes might have some conflicts with what I am doing now...

**Comment by tracyarciaga** [comment]
> > All changes are good. Approving this.
> 
> Hi @tracyarciaga , do you mind if you push that instead to my working branch `features/hdf-77-harvest-rewards`? I have a feeling that your changes might have some conflicts with what I am doing now...

I will just do a separate PR, cos it needs to be updated before the usability testing with users.  Thanks @schystz

File: src/components/VestingModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+interface VestingModalProps {
+  poolVestingInfo?: PoolVestingInfo
+}
+
+const VestingModal = ({ poolVestingInfo }: VestingModalProps) => {
+  const isOpen = useModalOpen(ApplicationModal.VESTING)
+  const toggleModal = useVestingModalToggle()
+  const poolName = poolVestingInfo?.name ?? 'pool'
+  const earningDSRT = poolVestingInfo?.balance.dsrt ?? 0,
+    earningHALO = poolVestingInfo?.balance.halo ?? 0
```
Comment (bitcoinbrisbane) [inline src/components/VestingModal/index.tsx]: Might read better on two lines IMO.

```suggestion
  const earningDSRT = poolVestingInfo?.balance.dsrt ?? 0
  const earningHALO = poolVestingInfo?.balance.halo ?? 0
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
One comment and need to fix the linting issues.

File: src/components/VestingModal/index.tsx (hunk 1 above)
Comment (schystz) [reply src/components/VestingModal/index.tsx]: Good catch. Will fix. Thanks @bitcoinbrisbane !

**Comment by schystz** [comment]
> One comment and need to fix the linting issues.

Thanks for the feedback @bitcoinbrisbane! Fixed both of them

**Review by ernestpascual — APPROVED** [review]

**Pull request merged**